registry/server/seeds/data/templates/master.html

```diff
@@ -57,6 +57,59 @@
 
     </style>
     <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+
+    <!-- ILC spinner stylesheets -->
```

Comment:
I don't think that we need spinner here. Out demo app People works a little bit incorrectly with it.